tools/accuracy_checker/accuracy_checker/adapters/mask_rcnn_with_text.py

```diff
@@ -0,0 +1,176 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+import cv2
+
+from .mask_rcnn import MaskRCNNAdapter
+from ..config import StringField, ConfigError, NumberField
+from ..representation import ContainerPrediction, TextDetectionPrediction
+from ..utils import contains_all
+
+
+class MaskRCNNWithTextAdapter(MaskRCNNAdapter):
+    __provider__ = 'mask_rcnn_with_text'
+
+    def __init__(self, launcher_config, label_map=None, output_blob=None):
+        super().__init__(launcher_config, label_map, output_blob)
+
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'classes_out': StringField(
+                description="Name of output layer with information about classes. "
+                            "(optional, if your model has detection_output layer as output).",
+                optional=True
+            ),
+            'scores_out': StringField(
+                description="Name of output layer with bbox scores."
+                            "(optional, if your model has detection_output layer as output).",
+                optional=True
+            ),
+            'boxes_out': StringField(
+                description="Name of output layer with bboxes."
+                            "(optional, if your model has detection_output layer as output).",
+                optional=True
+            ),
+            'raw_masks_out': StringField(
+                description='Name of output layer with raw instances masks'
+            ),
+            'texts_out': StringField(
+                description='Name of output layer with texts'
+            ),
+            'confidence_threshold': NumberField(
+                description='Confidence threshold that is used to filter out detected instances'
+            ),
+            'num_detections_out': StringField(
+                optional=True, description='Name of output layer with number valid detections '
+                                           '(used in MaskRCNN models trained with TF Object Detection API)'
+            ),
+            'detection_out': StringField(
```

Comment:
do you really need `detection_out` and `num_detections_out`?
MaskRCNN adapter is very common entity, where depends from source framework we have different outputs with respective processing.
I think you can stay only necessary for your model outputs.